### PR TITLE
change install to conda-forge

### PR DESCRIPTION
**Pull request by mpmdean**
This updates the docs in light of moving the build to conda-forge.

### CODE REVIEW

File: docs/source/user/installation.rst [hunk 1]
```diff
@@ -28,16 +28,16 @@ Then, create and activate a conda env, for example, called ``edrixs_env``::
 
 Install edrixs by::
 
-    conda install -c nsls2forge edrixs
+    conda install -c conda-forge edrixs
 
-where, `nsls2forge <https://anaconda.org/nsls2forge/>`_ is an Anaconda package repository maintained by `DAMA <https://github.com/nsls-ii-forge/edrixs-feedstock>`_ at NSLS-II at  Brookhaven National Laboratory. We endeavor to keep these up-to-date with our `tagged releases <https://github.com/NSLS-II/edrixs/releases>`_, but note that these builds will usually not correspond to the latest version of edrixs, which is available in the `master branch of edrixs <https://github.com/NSLS-II/edrixs>`_.
+We endeavor to keep the conda-forge release but to date, but note that these builds will usually not correspond to the latest version of edrixs, which is available in the `master branch of edrixs <https://github.com/NSLS-II/edrixs>`_.
```
Comment (mrakitin) [inline docs/source/user/installation.rst:33]:
```suggestion
We endeavor to keep the conda-forge release up to date, but note that these builds will usually not correspond to the latest version of edrixs, which is available in the `master branch of edrixs <https://github.com/NSLS-II/edrixs>`_.
```

**Review by mrakitin — APPROVED** [review]
Looks good to me; spotted a small typo.

**Comment by mrakitin** [comment]
Looks like there is a warning that is treated as an error. If you still want to proceed, you could comment out this line: https://github.com/NSLS-II/edrixs/blob/31f82573510ddfbc8424be13d4f79cac103364c3/docs/Makefile#L5

**Comment by mrakitin** [comment]
That's the error:
```
Warning, treated as error:
/home/runner/work/edrixs/edrixs/docs/source/user/quickstart.rst:5:undefined label: 'sphx_glr_auto_examples'
make: *** [Makefile:20: html] Error 2
make: Leaving directory '/home/runner/work/edrixs/edrixs/docs'
```

Looks like the label `sphx_glr_auto_examples` has to be defined. Is it supposed to point to another document?

**Comment by mrakitin** [comment]
> I believe the warning comes from the sphinx underline being longer than the title like this
> 
> ```
> Notes
> -------
> ```
That can be fixed too, but I don't think it's a problem, as other people used very long underlines for short titles. Per https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections, it cannot be shorter than a title.

**Comment by mrakitin** [comment]
https://github.com/sphinx-gallery/sphinx-gallery/issues/734 has a discussion and potential solutions to a similar issue.

**Comment by mpmdean** [comment]
I looked at the issues section of sphinx gallery following your suggestion. 

If I get the links as 
```
python -m sphinx.ext.intersphinx docs/build/html/objects.inv > links.txt
```

[links.txt](https://github.com/NSLS-II/edrixs/files/9700944/links.txt)

There does seem to be an entry `sphx_glr_auto_examples` so the problem is not transparent to me.
I think I'll just remove the line converting warnings to errors.

**Review by mrakitin — APPROVED** [review]

**Pull request merged**